less/index.less

```diff
@@ -56,4 +53,168 @@ table {
 html, body {
     height: 100%;
 }
+html{
+	font-size:62.5%;
+}
+p{
```

Comment:
instead of setting all these items to 1.6rem, I would recommend setting body { font-size: 1.6rem }, and then just introducing heading styles to override those when needed